src/po/README.txt

```diff
@@ -160,3 +160,20 @@ convert ja.po to EUC-JP (supposed as your system encoding):
 	"Content-Type: text/plain; charset=EUC-JP\n"
 
 There are examples in the Makefile for the conversions already supported.
+
+
+TRANSLATION OF VIM THE EDITOR PLUG-INS
+
+Vim supports displaying plugin messages for various native languages.
+Translation is available both for plugins that are supplied as part of the Vim
+editor (e.g. "optwin.vim") and for third-party plugin packages.
+
+To translate the plugins supplied with the Vim editor, you must specify a
+gettext() function call for the strings you want to translate.
+The translation of these strings will be retrieved by gettext() from the MO
+file "vim.mo".
+
+For third-party plugins, it is necessary to specify a once call to the
+bindtextdomain() function in scripts containing translation strings, and for
+all message strings, the gettext() function. For more information, 
```

Comment:
the gettext() function -> to add a {package} argument to the gettext() function

Reply:
Thank you, I have corrected it.